app/uk/gov/hmrc/bindingtariffclassification/repository/SearchMapper.scala

```diff
@@ -31,8 +31,8 @@ class SearchMapper {
     JsObject(
       Map() ++
         filter.applicationType.map(s => "application.type" -> JsString(s.toString)) ++
-        filter.queueId.map("queueId" -> nullifyNoneValues(_)) ++
-        filter.assigneeId.map("assignee.id" -> nullifyNoneValues(_)) ++
+        filter.queueId.map("queueId" -> mappingNoneOrSome(_)) ++
```

Comment:
Figured might as well do the same for Queue rather than create a second mapping method